task_interpreter_ipf_client/i18n/sv_SE.po

```diff
@@ -0,0 +1,309 @@
+msgid "Response Code"
+msgstr "Svart Kod"
+
+#. module: task_interpreter_ipf_client
+#: model:ir.model.fields,field_description:task_interpreter_ipf_client.field_ipf_interpreter_request_history__response_headers
+msgid "Response Headers"
+msgstr "Svars huvud"
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Server Environment."
+msgstr "Server Miljö"
+
+#. module: task_interpreter_ipf_client
+#: model:ir.model.fields,field_description:task_interpreter_ipf_client.field_res_config_settings__ipf_server_url
+msgid "Server URL"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: selection:res.config.settings,ipf_environment:0
+msgid "T2"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: model:ir.model,name:task_interpreter_ipf_client.model_ipf_interpreter_client
+msgid "Task Interpreter IPF client"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: selection:res.config.settings,ipf_environment:0
+msgid "U1"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Update Intepreter gender preferences"
+msgstr "Uppdatera Tolkkönspreferenser"
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Update Intepreter languages"
+msgstr "Uppdatera Tolk språk"
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Update Intepreter remote types"
+msgstr "Uppdatera Distanstolks typer "
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Update Intepreter types"
+msgstr "Uppdatera Tolk varianter"
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Update all Intepreter data"
+msgstr "Uppdatera alla Tolk datalager"
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.res_config_settings_view_form
+msgid "Update data"
+msgstr "Uppdatera data"
+
+#. module: task_interpreter_ipf_client
+#: model:ir.model.fields,field_description:task_interpreter_ipf_client.field_ipf_interpreter_client_test__url
+#: model:ir.model.fields,field_description:task_interpreter_ipf_client.field_ipf_interpreter_request_history__url
+msgid "Url"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.ipf_client_config_view_form
+msgid "distanstolktyp"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.ipf_client_config_view_form
+msgid "kon"
```

Comment:
Should this and the terms below not be in english and then translated?

Reply:
Non trivial to fix now, will be fixed in the future